python/pycylon/data/table.pyx

```diff
@@ -2366,7 +2365,7 @@ cdef class Table:
         return PyLocIndexer(self, "iloc")
 
     @staticmethod
-    def concat(tables: List[Table], axis: int = 0, join: str = 'inner', algorithm: str = 'sort', distributed=False):
+    def concat(tables: List[Table], axis: int = 0, join: str = 'inner', algorithm: str = 'sort'):
```

Comment:
A local concat operation is needed even in the distributed setting. 
Shall we add separate methods for local and distributed or we can keep this flag?

Reply:
Note: This is used as it is in the UNO application

Reply:
Oh. I see your point. Ishall we add a `distributed_concat` to the table API (like in other operations)?

Reply:
Yeah that sounds great :+1:

Reply:
great. I'll add that now. And the use it in dataframe api accordingly.